uisdk/src/main/java/com/karhoo/uisdk/KarhooUISDK.kt

```diff
@@ -46,5 +47,12 @@ object KarhooUISDK {
     fun setConfiguration(configuration: KarhooUISDKConfiguration) {
         KarhooUISDKConfigurationProvider.setConfig(configuration)
         KarhooApi.setConfiguration(configuration)
+
+        cacheVehicleMappings()
+    }
+
+    private fun cacheVehicleMappings() {
+        VehicleMappingsProvider.setup(KarhooApi.quotesService)
+        VehicleMappingsProvider.retrieveVehicleMappings()
```

Comment:
The mappings need to be refreshed periodically. The config cache max-age is 1 hour. Maybe refresh the cache every time the booking flow is started? Should be enough